cxx-api-examples/punctuation-cxx-api.cc

```diff
@@ -0,0 +1,50 @@
+// cxx-api-examples/fire-red-asr-cxx-api.cc
+// Copyright (c)  2025  Xiaomi Corporation
+
+//
+// This file demonstrates how to use FireRedAsr AED with sherpa-onnx's C++ API.
```

Comment:
Please update the comment or you can delete it since you have already included in the usage info in the code.